packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/DepositMethods/BankDeposit/index.tsx

```diff
@@ -59,12 +59,32 @@ const BankDeposit = ({ icon, onClick, value }: Props) => (
             />
           </DisplayTitleBank>
           <DisplaySubTitle>
-            <FormattedMessage id='copy.instantly_available' defaultMessage='Instantly Available' />
+            {value.currency === 'GBP' ? (
+              <FormattedMessage
+                id='copy.arrive_instantly'
+                defaultMessage='Should arrive instantly'
+              />
+            ) : value.currency === 'EUR' ? (
+              <FormattedMessage
+                id='copy.arrive_instantly'
+                defaultMessage='Should arrive instantly'
+              />
+            ) : value.currency === 'USD' ? (
+              <FormattedMessage
+                id='copy.arrive_in_3_business_days'
+                defaultMessage='Should arrive in 3 business days'
+              />
+            ) : (
+              <FormattedMessage
+                id='copy.instantly_available'
+                defaultMessage='Instantly Available'
+              />
+            )}
```

Comment:
What about doing this in a helper function outside? It's clearly a `switch`, maybe with a placeholder value to avoid the component repetition. Also GBP and EUR seem to be the same.

```javascript
const helper = (currency) => {
  switch(currency):
    case 'GBP':
    case 'EUR':
      return  {id: 'copy.instantly_available',  defaultMessage:'Should arrive instantly'};
    ...
    default:
      return  {id: 'copy.instantly_available',  defaultMessage:'Instantly Available'};
}

And replace all this with <FormattedMessage {...helper(currency)} />
```